runtime/doc/starting.txt

```diff
@@ -1087,7 +1089,7 @@ Avoiding trojan horses ~
 							*trojan-horse*
 While reading the "vimrc" or the "exrc" file in the current directory, some
 commands can be disabled for security reasons by setting the 'secure' option.
-This is always done when executing the command from a tags file.  Otherwise it
+This is always done when executing the command from a tags file.  If not, it
```

Comment:
here I find the Otherwise more clear

Reply:
Sure, I'll put it back.  The reason I went for "If not," is 
that I lothed to reformat to width the lines after for the  
needed change "Otherwise,".